sw/airborne/firmwares/rotorcraft/autopilot_static.c

```diff
@@ -173,8 +173,14 @@ void autopilot_static_periodic(void)
     SetCommands(commands_failsafe);
   } else {
     guidance_v_run(autopilot_in_flight);
-    guidance_h_run(autopilot_in_flight);
+    guidance_h_run(autopilot_in_flight);    
     SetRotorcraftCommands(stabilization_cmd, autopilot_in_flight, autopilot_motors_on);
+
+    if (lock_wings == 1 && autopilot_motors_on && commands[COMMAND_THRUST] > 0) {
```

Comment:
I think this should either go above (set `stabilization_cmd` with the proper value in a control loop) or below (as part of the actuators mixing defined in the airframe file) in the autopilot architecture.